features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/OspfElementDao.java

```diff
@@ -46,6 +46,12 @@
 
     public List<OspfElement> findAllByRouterId(InetAddress routerId);
 
+    /**
+     * Returns all OspfElements that a have an ospfRouterId that matches an ospfRenRouterId of an OspfLink related to the given
```

Comment:
Maybe a typo: Isn't it ospfRemRouterId, Rem for remote?

Reply:
fixed